lib/rules/await-async-query.ts

```diff
@@ -1,41 +1,21 @@
 import { ESLintUtils, TSESTree } from '@typescript-eslint/experimental-utils';
-import { getDocsUrl } from '../utils';
+import { getDocsUrl, LIBRARY_MODULES } from '../utils';
```

Comment:
I think there's an improvement to do on the `LIBRARY_MODULES` naming. Indeed, it doesn't make sense to make the rule fire on the async queries for `@testing-library/cypress`. Right now it works because it's not included in the `LIBRARY_MODULES` constants, however it's still a library module.